### ISSUE TITLE
Slide order: -1 when carousel has only one item or after element was added in carousel.

### ISSUE THREAD

**Issue by slobodadev** [ismail9k/vue3-carousel#37]
My template:
```
<carousel v-if="presentedVideos.length"  :wrap-around="true">
  <slide class="slide" v-for="video in presentedVideos" :key="video.id">
     <iframe class="videoFile" :src="video.video_link"></iframe>
  </slide>
  <template #addons>
    <navigation />
  </template>
</carousel>
```
1. When `presentedVideos` ref contains only one item in array - carousel has crashed. I can't slide anymore.
![Screenshot from 2021-04-05 15-12-42](https://user-images.githubusercontent.com/55140508/113687840-69113a00-96d1-11eb-8bb8-9df574693f18.png)
Element outside 

2. Same problem when I add new instance.
![Screenshot from 2021-04-06 12-18-16](https://user-images.githubusercontent.com/55140508/113688838-5814f880-96d2-11eb-9ba6-efa19dd9ceed.png)
(after reload problem disappears)

3. Another case when I remove instance. I'm using this way -  https://github.com/ismail9k/vue3-carousel/issues/36#issuecomment-813019681
Carousel items is not updating immediately. I should click on the slide arrows. After that removed instance is disappeared.

**Comment by ismail9k** [comment]
Could you provide me with a working [sandbox](https://codesandbox.io) example, so I can debug this issue :)

**Comment by mcorcias** [comment]
when the carousel has only one item see nothing, after you update data no see immediately the changes

**Comment by ismail9k** [comment]
fixed in `v0.1.15`

**Comment by slobodadev** [comment]
@ismail9k  Still exits a problem with dynamic data. When you remove an instance, the changes is not appearing immediately. You should refresh the page. Even a bad solution with forceUpdate doesn't work
